docs/usage/configuration-options.md

```diff
@@ -1679,6 +1684,30 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
+
+Use `matchCategories` to restrict rules to a particular language or group.
+The categories can be found in the [manager documentation](./modules/manager.md).
+
+<!-- prettier-ignore -->
+!!! warning
+    Rules with `matchCategories` are only applied _after_ extraction of a repository, therefore it can not be used to completely prevent analyzes of a repository.
```

Comment:
This sentence feels too long for me. I think we can split it up, and find better words for:

> therefore it can not be used to completely prevent analyzes of a repository.

I think we mean something like this:

```suggestion
    Rules with `matchCategories` are only applied _after_ extraction of a repository.
    You can not use `matchCategories` to stop processing a repository.
```

Reply:
Sounds good to me!